spec/controllers/pages_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe PagesController, type: :controller do
+
+  describe 'GET #home' do
+    context 'when viewing homepage in the browser' do
+      it 'returns a 200 OK response' do
+        (expect(response.status).to eq(200))
```

Comment:
Few things here. Firstly, we're not actually calling the endpoint in this test. In order to call the endpoint you need to add `get :home` on the line before our expect. Now in testing this myself I came across issues running the tests. Apparently RSpec is still in beta for supporting Rails 6. So if you want to use it you'll have to adjust the rspec-rails dependency in your Gemfile to be `gem 'rspec-rails', '~> 4.0.0.beta4'`. We also don't really need the surrounding brackets on the expect statement.